src/parser/core/Events.ts

```diff
@@ -330,6 +331,8 @@ export interface Event<T extends string> {
   prepull?: boolean;
   /** Other events associated with this event. Added by WoWA normalizers. Meaning is context sensitive */
   _linkedEvents?: LinkedEvent[];
+  /** Set of eventLinks that have been processed already to avoid duplicated linking */
+  _processedLinks?: Set<EventLink>;
```

Comment:
i am a little torn on this being a `Set<EventLink>` vs a `Set<String>`. relying on object identity has some awkwardness around object destructuring / spreading, but i don't think people generally use that here? and it would require some very weird call sequence to have the same exact normalizer (not two unique files, but one file) run twice in the first place

Reply:
I think using EventLink is likely more precise because you can have 2 different event links with the same `linkRelation` but different `reverseLinkRelation` which means you would need 2 strings in order to uniquely identify it.

Reply:
that is reasonable